src/Equations/anisotropic/Model/datastructures.hpp

```diff
@@ -72,6 +72,14 @@ namespace seissol {
       double c56;
       double c66;
 
+      double getLambda() const override {
+        return (c11 + c22 + c33) / 3.0 - 2.0*getMu();
+      }
```

Comment:
These are used only for the Free Surface Boundary condition in the end, right? If that is the case, wouldn't it be better if we just write that that surface boundary condition is not yet implemented for anisotropic material? Mainly because I am not really sure if the free surface boundary condition just behaves this way with anisotropic material or if there is more math involved for the same. 

We are also yet to solve the acoustic-anisotropic coupling problem and this could then be taken care of there...

Reply:
it seems getLambda is required to compute P wave speed, which is needed to get the max wave speed and therefore the min dt.